rcode-reader/src/main/java/eu/rssw/pct/AccessType.java

```diff
@@ -0,0 +1,39 @@
+package eu.rssw.pct;
+
+import java.util.EnumSet;
+import java.util.Set;
+
+public enum AccessType {
+  PUBLIC,
+  PRIVATE,
+  PROTECTED,
+  STATIC,
+  ABSTRACT,
+  FINAL,
+  CONSTRUCTOR;
+
+  public static Set<AccessType> getTypeFromString(int val) {
+    Set<AccessType> set = EnumSet.noneOf(AccessType.class);
+    switch (val & 0x07) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a default case to this switch. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)